tests/charm++/jacobi3d/jacobi3d.C

```diff
@@ -137,7 +142,17 @@ class Main : public CBase_Main {
 		blockDimZ = atoi(m->argv[6]);
 		maxIter = atoi(m->argv[7]);
 		ckptFreq = atoi(m->argv[8]);
-	}
+	} else if (m->argc == 10) {
+                arrayDimX = atoi(m->argv[1]);
+                arrayDimY = atoi(m->argv[2]);
+                arrayDimZ = atoi(m->argv[3]);
+                blockDimX = atoi(m->argv[4]);
+                blockDimY = atoi(m->argv[5]);
+                blockDimZ = atoi(m->argv[6]);
+                maxIter = atoi(m->argv[7]);
+                ckptFreq = atoi(m->argv[8]);
+                overhead = atoi(m->argv[9]);
```

Comment:
Should `overhead` be 7 instead of 9? In the other case, the optional `maxIter` and `ckptFreq` parameters come at the end, so it's a little inconsistent for them not to be here.

Reply:
This is mapped correctly, the corresponding print statements before this is where this change has not been updated hence the confusion. Will update that in the next commit.